### DISCUSSION TITLE
Discord is dead

### DISCUSSION THREAD

**Discussion by h0ek** [hexojs/hexo#5547] [General]
Hexo has several communication channels like Discussions, Google Group, Discord, Gitter and Telegram. I don't know about the other channels, but Discord is dead. I've been there a long time and the server looks abandoned. Sometimes a few people reply, I try to help there when I know how, but in most cases someone comes, asks, no one replies and after a few weeks someone may or may not reply. I like the idea of discussions here on GitHub, I learned about it myself today while browsing GitHub issues related to communication and discord. 

Where I'm going with this is that I think it would be useful to standardise communication between users and between users and developers. Focus on one main communication channel that would bring us all together and build a strong community that can support each other. Currently there are several communication channels listed and each one seems to be more or less abandoned or not properly maintained. Hexo is a great solution, but its less technical users wander between different sites looking for support or answers to simple questions. 

You may want to consider an official communication channel for the Hexo community. Having a strong community behind a project, easily accessible to both old and new users, could become another asset for the project. Often the quality of community support is one of the criteria for choosing a particular open source solution. GitHub is great for code changes and bug reporting. But it's not very good for casual chat about the project. I know there are so many users, so many ideas, IRC, Matrix, Telegram, etc. But if there was a place designated by the developers and they were there, I think it would work for everyone to get their act together. 

Discord/Matrix is a good tool for that. You can set up multiple channels like support, ideas, chitchat, releases, bugs, feedback, help, improvements, off-topic, etc. and dedicated language groups and integrate it with GitHub and manage permissions for specific chats. Just an idea. 

Personally, when I started with Hexo many years ago, I mainly used the documentation, learned from my mistakes or searched for answers on Google, where other projects bring users together in one place and facilitate discussion in a dedicated group where someone has always had a problem or knows a simple solution. 

I hope this makes sense. But if I am wrong or missing something, forget the whole idea :D

**Comment by stevenjoezhang** [comment]
The main discussion community for Hexo is still on GitHub (through Issues and Discussions). For instant messaging platforms like Discord, the activity level may not be as high as other types of groups, such as chat groups for gaming. There could be several reasons for this: 
1. Users usually initiate discussions when they encounter problems, but such issues are not frequent. More often, once the environment is set up correctly, users can easily write and update their blogs themselves.
2. Now there are more ways to get help, such as through ChatGPT. Although large language models might not have as detailed knowledge about Hexo as the maintainers do, they are already sufficient to solve common problems and provide faster feedback.
3. For feature requests, there might be a longer time cycle, which does not align well with the immediacy of instant messaging chat platforms. Developers often need time to write code and test. Moreover, discussions that take place in chat groups instead of on GitHub can lead to issues that are hard to track (e.g., when wanting to cross-reference multiple bug reports and pull requests).

In the future, we could focus more on specific chat channels and strive to provide more timely feedback on the messages from the community.
